docs/src/main/sphinx/release/release-436.md

```diff
@@ -0,0 +1,52 @@
+# Release 436 (10 Jan 2024)
+
+## General
+
+* {{breaking}} Require JDK 21.0.1 to run Trino. ({issue}`20010`)
+* Improve query plans by not generating redundant predicates. ({issue}`16520`)
```

Comment:
Should this be reworded as performance improvement or is there some other benefit as well?

Reply:
Yes, it's a performance improvement.